src/pages/workspace/expensifyCard/WorkspaceCardsListLabel.tsx

```diff
@@ -0,0 +1,132 @@
+import type {RouteProp} from '@react-navigation/native';
+import {useRoute} from '@react-navigation/native';
+import React, {useEffect, useMemo, useRef, useState} from 'react';
+import {View} from 'react-native';
+import type {StyleProp, ViewStyle} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import Button from '@components/Button';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import Popover from '@components/Popover';
+import {PressableWithFeedback} from '@components/Pressable';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import * as CurrencyUtils from '@libs/CurrencyUtils';
+import getClickedTargetLocation from '@libs/getClickedTargetLocation';
+import type {FullScreenNavigatorParamList} from '@navigation/types';
+import variables from '@styles/variables';
+import * as Report from '@userActions/Report';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type SCREENS from '@src/SCREENS';
+
+type WorkspaceCardsListLabelProps = {
+    /** Label type */
+    type: ValueOf<typeof CONST.WORKSPACE_CARDS_LIST_LABEL_TYPE>;
+
+    /** Label value */
+    value: number;
+
+    /** Additional style props */
+    style?: StyleProp<ViewStyle>;
+};
+
+function WorkspaceCardsListLabel({type, value, style}: WorkspaceCardsListLabelProps) {
+    const route = useRoute<RouteProp<FullScreenNavigatorParamList, typeof SCREENS.WORKSPACE.EXPENSIFY_CARD>>();
+    const [policy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${route.params.policyID}`);
+    const styles = useThemeStyles();
+    const {windowWidth} = useWindowDimensions();
+    const {shouldUseNarrowLayout} = useResponsiveLayout();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+    const [bankAccountList] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST);
+    const [isVisible, setVisible] = useState(false);
+    const [anchorPosition, setAnchorPosition] = useState({top: 0, left: 0});
+    const anchorRef = useRef(null);
+
+    const policyCurrency = useMemo(() => policy?.outputCurrency ?? CONST.CURRENCY.USD, [policy]);
+    const isConnectedWithPlaid = useMemo(() => !!Object.values(bankAccountList ?? {})[0]?.accountData?.additionalData?.plaidAccountID, [bankAccountList]);
```

Comment:
We don't want to get the first bank account that is connected to Plaid. We want to check if the bank account that is configured as the settlement bank account for the Expensify Card on the workspace is connected to Plaid.

So we'll need to get the settlementBankAccountID from the private_expensifyCardSettings NVP and check if that is connected via Plaid.

Reply:
@MariaHCD can we do that in a follow-up when nvp_private_expensifyCardSettings is ready? I will add a comment here

Reply:
Sounds good to me 👍🏼